doc/manual/cookbooks/benchmarks/burstedde/burstedde.prm

```diff
@@ -27,3 +26,9 @@ end
 subsection Postprocess
   set List of postprocessors = visualization, velocity statistics, BursteddePostprocessor
 end
+
+subsection Solver parameters
+  subsection Stokes solver parameters
+    set Number of cheap Stokes solver steps = 0
+  end
+end
```

Comment:
If this was not in the original file, you don't have to add this here (in particular as there's no documentation on why the parameter is chosen like this).